docs/release-notes/1.1.0/release-1.1.0.md

```diff
@@ -0,0 +1,105 @@
+
+## **Bug Fixes**
+### Serious
+- **Math Kernel Library fails to load with latest libomp:** Fixed by
+  [PR#3721](https://github.com/dotnet/machinelearning/pull/3721) this bug made
+  it impossible for anyone to check code into master branch because it was
+  causing build failures.
+
+- **Transform Wrapper fails at deserialization:** Fixed by
+  [PR#3700](https://github.com/dotnet/machinelearning/pull/3700) this bug
+  affected first party(1P) customer. A model trained models using
+  [NimbusML](https://github.com/microsoft/NimbusML)(Python bindings for
+  [ML.NET](http://dot.net/ml)) and then loaded for scoring/inferencing using
+  ML.NET will hit this bug. 
+
+- **Index out of bounds exception in KeyToVector transformer:** Fixed by
+  [PR#3763](https://github.com/dotnet/machinelearning/pull/3763) this bug closes
+  following github issues:
+  [3757](https://github.com/dotnet/machinelearning/issues/3757),[1751](https://github.com/dotnet/machinelearning/issues/1751),[2678](https://github.com/dotnet/machinelearning/issues/2678).
+  It affected first party customer and also github users. 
+
+### Other
+- Download images only when not present on disk and print warning messages when
+  converting unsupported pixel format by
+  [PR#3625](https://github.com/dotnet/machinelearning/pull/3625)
+- [ML.NET](http://dot.net/ml) source code does not build in VS2019 by
+  [PR#3742](https://github.com/dotnet/machinelearning/pull/3742)
+- Fix SoftMax precision by utilizing double in the internal calculations by
+  [PR#3676](https://github.com/dotnet/machinelearning/pull/3676)
+- Fix to the official build due to API Compat tool change by
+  [PR#3667](https://github.com/dotnet/machinelearning/pull/3667)
+
+## **Breaking Changes**
+None
+
+## **Enhancements**
+- API Compat tool by
+  [PR#3623](https://github.com/dotnet/machinelearning/pull/3623) ensures future
+  changes to ML.NET will not break the stable API released in 1.0.0.
+- Upgrade the TensorFlow version from 1.12.0 to 1.13.1 by
+  [PR#3758](https://github.com/dotnet/machinelearning/pull/3758)
+- API for saving time series model to stream by
+  [PR#3805](https://github.com/dotnet/machinelearning/pull/3805)
+
+## **Documentation and Samples**
+- L1-norm and L2-norm regularization documentation by
+  [PR#3586](https://github.com/dotnet/machinelearning/pull/3586)
+- Sample for data save and load from text and binary files by
+  [PR#3745](https://github.com/dotnet/machinelearning/pull/3745)
+- Sample for LoadFromEnumerable with a SchemaDefinition by
+  [PR#3696](https://github.com/dotnet/machinelearning/pull/3696)
+- Sample for LogLossPerClass metric for multiclass trainers by
+  [PR#3724](https://github.com/dotnet/machinelearning/pull/3724)
+- Sample for WithOnFitDelegate by
+  [PR#3738](https://github.com/dotnet/machinelearning/pull/3738)
+
+## **Remarks**
+- [Microsoft.ML.TensorFlow](https://www.nuget.org/packages/Microsoft.ML.TensorFlow/)
+  has been upgraded from 0.12(preview) to 1.0(release).
```

Comment:
Shouldn't the Microsoft.ML.TensorFlow package move to v1.1 (instead of v1.0) when going GA so it is aligned to the ML.NET version in the same timeframe?

Reply:
You are replying to an outdated code. We are not GA'ing tensorflow this release.